packages/shared-mocks-lib/src/repo/settlement_matrix_request_repo_mock.ts

```diff
@@ -60,15 +62,43 @@ export class SettlementMatrixRequestRepoMock implements ISettlementMatrixRequest
 		return Promise.resolve(null);
 	}
 
-	async getMatrices(state?:string): Promise<ISettlementMatrix[]>{
-		const ret: ISettlementMatrix[] = [];
+	async getMatrices(
```

Comment:
In these methods, because we call them in many places in the aggragte, we need the default to be return all (without pagination)